website/index.js

```diff
@@ -1,4 +1,5 @@
-require('dotenv').config();
```

Comment:
What about the `.env` file for the other configs like cloudinary keys?

Reply:
Not sure what you mean? All the env vars should be in the `config.env`, as they are in dev. (Just the file is in a different spot in production)